lib/dependencies.sh

```diff
@@ -31,14 +31,16 @@ run_if_present() {
 
 yarn_node_modules() {
   local build_dir=${1:-}
+  $NODE_VERBOSE && local opt_verbose='--verbose' || local opt_verbose=''
```

Comment:
I agree in JS. Perhaps I'm not used to reading idiomatic bash code yet, but I'd much prefer just a simple if / else in this case.